### ISSUE TITLE
Moving head around makes image to become blurry

### ISSUE THREAD

**Issue by LiberQuack** [wheaney/breezy-desktop#20]
I noticed that when moving my head around, text become blurry, a little bit similar when motion blur is applied on games

I guess it's something related to shader, I'd love helping you by opening a PR, but I don't understand that much of shaders and gnome as of now to give you a hand

Anyways, amazing work!!!
For xreal glasses the experience is already better than on Windows :raised_hands:

**Comment by wheaney** [comment]
Not really blurry, it's actually a doubling of frames, you can especially see it if you move your head in a very slight circular motion (look up, right, down, left pretty swiftly) and look at something with high contrast like text, you'll get two images floating around each other. I've played around A LOT with this, and actually found it to be an inherent property of the way that the displays in the glasses refresh. If you disable my plugin and find a way to duplicate that test in-game using camera movements (controlling with a mouse makes it easy to quickly move in a circular motion), you'll see this same doubling.

So, long story short, this is a hardware issue with the displays in the glasses, and there's nothing I've found on my end that can be done to fix it.
